common/stats/stats.go

```diff
@@ -30,6 +30,8 @@ import (
 // For testing.
 var Time StatsTime = DefaultStatsTime()
 
+type UpTimeReportIntvl time.Duration
```

Comment:
Seems odd to make a type for this when it looks like all uses are converting directly from or directly to a time.Duration